content/media-production-on-pop.md

```diff
@@ -89,8 +90,17 @@ Install Directly From the Pop!\_Shop
 
 Krita is a free and open source raster graphics editor that is primarily focused on digital drawing and painting. It features an OpenGL-accelerated canvas, non-destructive layers and masks, an advanced brush engine, and support for many drawing tablets.
 
+Install Directly From the Pop!_Shop
+
 ### Flowblade
 
 Flowblade is a multitrack non-linear video editor with a set of industry standard editing tools suitable for beginners and masters alike to help make your vision a reality of image and sound.
 
-Install Directly From the Pop!\_Shop
+Install Directly From the Pop!_Shop
+
+### Shotcut 
+
+Shotcut is a free, open source, cross-platform video editor. Edit your videos with this lightweight editor.
```

Comment:
`lightweight` is really the only new word that the second sentence is adding here.

Reply:
I've replaced the second sentence with one that adds a little more value.